app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,81 @@
+<h1 class="w-fit text-3xl font-semibold ml-[40%] my-6 underline underline-offset-1"><%= @recipe.name %></h1>
+<div class="w-4/5 ml-[5%]">
+  <div class="flex justify-between items-center">
+    <span class="text-2xl font-medium">Preparation time: <%= @recipe.preparation_time %> minutes</span>
+    <% if @recipe.user == current_user %>
+      <%= form_with model:@recipe, url: recipes_path  do |form| %>
+        <label class="relative inline-flex items-center cursor-pointer">
+          <%= form.check_box :public, class: 'sr-only peer', id: 'public-toggle' %>
+          <div class="w-11 h-6 bg-gray-200 peer-focus:outline-none peer-focus:ring-4 peer-focus:ring-red-300 rounded-full peer peer-checked:after:translate-x-full peer-checked:after:border-white after:content-[''] after:absolute after:top-[2px] after:left-[2px] after:bg-white after:border-gray-300 after:border after:rounded-full after:h-5 after:w-5 after:transition-all peer-checked:bg-red-600"></div>
+          <span class="ml-3 text-sm font-semibold flex items-center">Public</span>
+        </label>
+      <% end %>
+    <% end %>
+  </div>
+  <span class="text-2xl font-medium">Cooking time: <%= @recipe.cooking_time %> minutes</span>
+  <div class="flex items-center justify-between mt-4">
+    <p class="text-lg"><%= @recipe.description %></p>
+    <% if @recipe.user == current_user %>
+      <%= link_to 'Add ingredient', new_user_recipe_recipe_food_path(recipe_id: @recipe, user_id: @recipe.user), class:"bg-yellow-300 py-2 px-4 rounded-full" %>
+    <% end %>
+  </div>
+</div>
+<hr class="border-[1px] my-8 ml-[5%] w-11/12 border-red-500">
+<h2 class="w-fit text-3xl font-semibold ml-[40%] my-6 underline underline-offset-1">Ingredients</h2>
+<table class="table ml-[5%] w-11/12">
+  <thead >
+    <tr>
+      <th>Name</th>
+      <th>Measurment unit</th>
+      <th>Price</th>
+      <th>Actions</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @recipe.recipe_foods.each_with_index do |recipe_food, index| %>
+      <tr class="<%= index.even? ? 'bg-red-300' : 'bg-transparent'%>">
+        <td><%= recipe_food.food.name %></td>
+        <td><%= recipe_food.quantity %> <%= recipe_food.food.measurement_unit %></td>
+        <td>$<%= recipe_food.value %></td>
+        <td>
+          <% if @recipe.user == current_user %>
+           <%= button_to 'Delete Ingredient', user_recipe_recipe_food_path(recipe_id: @recipe, user_id: @recipe.user, id: recipe_food), method: :delete, data: { confirm: 'Are you sure?' }, class: 'bg-red-500 py-2 px-4 rounded-full' %>
```

Comment:
- [ ] Also, you may please reference the video attached, in the wireframe which happens to be your guide, I think we have 2 action buttons here `Modify & Remove` :pray: Kindly follow this minimalistic design guide :wink: